### PR TITLE
add more text to README

### PR DESCRIPTION
**Pull request by clairefields15**
I added some more text to see what this end of the process is like.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 Hi this is my readme!
 Here's my addition to read me.
 Here's another addition to test what the hell we're doing.
+
+Even more text! Omg look at us go!
```
Comment (lschnell8) [inline README.md:5]: We're killin it!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+Wow.
```
Comment (lschnell8) [inline README.md:7]: Equally astonished!

File: README.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+
+Much edits.
```
Comment (lschnell8) [inline README.md]: Maybe 'more edits' or 'lots of edits' here instead? Let's discuss.

**Review by lschnell8 — CHANGES_REQUESTED** [review]
Looking good!

File: README.md (hunk 3 above)
Comment (clairefields15) [reply README.md]: Changed it!

**Review by lschnell8 — APPROVED** [review]

**Pull request merged**